libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -140,6 +140,7 @@ void SRV_Channel::aux_servo_function_setup(void)
     case k_flap:
     case k_flap_auto:
     case k_egg_drop:
+    case k_lift_release:
         set_range(100);
```

Comment:
I would like to start setting range/angle of 1 for new things.
```suggestion
    case k_lift_release:
        set_range(1);
```
Eventually we can get rid of all the `* 4500`

Reply:
there are no set_range(1) calls in master right now, and while the range is a uint16_t I don't think 1 is a good choice as a default as it means it can only be binary. That is OK for this case, but not a good pattern